imap/caldav_alarm.c

```diff
@@ -632,7 +632,7 @@ static int read_lastalarm(struct mailbox *mailbox,
                             annotname, "", &annot_buf);
 
     if (annot_buf.len &&
-        sscanf(buf_cstring(&annot_buf), "%ld %ld",
+        sscanf(buf_cstring(&annot_buf), TIME_T_FMT " " TIME_T_FMT,
```

Comment:
Hmmm, does it make sense to use these format specifiers for `sscanf` conversions?  I'm not sure.  @ksmurchison, any thoughts?

Reply:
Clang on OpenBSD was complaining:

```
imap/caldav_alarm.c:639:16: warning: format specifies type 'long *' but the argument has type
   'time_t *' (aka 'long long *') [-Wformat]
               &data->lastrun, &data->nextcheck) == 2) {
               ^~~~~~~~~~~~~~
imap/caldav_alarm.c:639:32: warning: format specifies type 'long *' but the argument has type
   'time_t *' (aka 'long long *') [-Wformat]
               &data->lastrun, &data->nextcheck) == 2) {                                                                                                                                                                                                                                                                
                               ^~~~~~~~~~~~~~~~
```